defaults/CLog.cmake

```diff
@@ -38,10 +43,20 @@ function(CLOG_GENERATE_TARGET)
         list(APPEND clogfiles ${ARG_CLOG_C_FILE})
     endforeach()
 
+
+

```

Comment:
Do you have the same newline disease as Chris?

Reply:
Yes 👍